dev/com.ibm.ws.security.acme/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -80,3 +80,12 @@ preferCRLs.desc=By default, OCSP is the preferred mechanism for checking revocat
 
 disableFallback=No fallback
 disableFallback.desc=Disable the certificate revocation status checker fallback mechanism. During certificate revocation status check, the default behavior is fallback and check by using the non-preferred mechanism (OCSP or CRLs). Default is false.
+
+certCheckerSchedule=Certificate checker schedule
+certCheckerSchedule.desc=Time period for the ACME CA service to check if the certificate is expiring or revoked. If expiring or revoked, the certificate is renewed. If the first renew request does not work, the certificate renewal request continues until a new certificate is received using the timing set by certCheckerErrorSchedule.  Setting the certCheckerSchedule property to zero or a negative value disables automatic certificate renewal.
+
+certCheckerErrorSchedule=Certificate checker error schedule
+certCheckerErrorSchedule.desc=Time period for the ACME CA service to check if the certificate is expiring or revoked if the first request failed. If expiring or revoked, the certificate is renewed.
```

Comment:
does this specify an interval at which to issue renew requests or a span of time during which renewal requests occur (and beyond which they stop?)

Reply:
The interval on which we check if we need to do a renew request. I think I didn't really finish doing this description well. I will add to it and you can give it another eyeball.